docs/src/modules/java/pages/spring-client.adoc

```diff
@@ -0,0 +1,293 @@
+
+== How to get started
+
+To get started, do the following:
+
+- https://github.com/lightbend/akkaserverless-java-sdk/archive/refs/heads/main.zip[Download] and unzip the source repository for this guide, or clone it using Git:
+
+`git clone https://github.com/lightbend/akkaserverless-java-sdk.git`
+
+- cd into `akkaserverless-java-sdk/samples/java-valueentity-counter`
+
+== Build and Run the Akka Serverless Service Locally
+
+With the source repository downloaded, you can build and run the Akka Serverless Service locally. 
+
+Run the Akka Serverless Service using Maven:
+
+`mvn compile exec:exec`
+
+You should see output similar to the following:
+
+```
+[INFO] --- exec-maven-plugin:3.0.0:exec (default-cli) @ valueentity-counter ---
+{"timestamp":"2022-03-02T23:38:12.174Z","thread":"main","logger":"com.example.Main","message":"starting the Akka Serverless service","context":"default","severity":"INFO"}
+{"timestamp":"2022-03-02T23:38:12.829Z","thread":"akkaserverless-akka.actor.default-dispatcher-2","logger":"akka.event.slf4j.Slf4jLogger","message":"Slf4jLogger started","context":"default","severity":"INFO"}
+```
+
+== Build and Run the Akka Serverless proxy locally
+
+In another terminal window, run the Akka Serverless proxy using Docker:
+
+`docker compose up`
+
+You should see output similar to the following:app-name: 
+
+```
+akka-serverless-proxy_1   | {"timestamp":"2022-03-02T23:40:40.766Z","mdc":{"akkaAddress":"akka://akkaserverless-proxy@172.21.0.2:25520","akkaSource":"akka://akkaserverless-proxy/user/discovery-manager","sourceActorSystem":"akkaserverless-proxy"},"logger":"com.akkaserverless.proxy.DiscoveryManager","message":"gRPC proxy started at 0.0.0.0:9000","severity":"INFO","thread":"akkaserverless-proxy-akka.actor.default-dispatcher-8"}
+```
+
+== Building Spring REST Client
+
+The Spring REST Client would enable Spring application to integrate with REST endpoints exposed via Akka Serverless Service.
+
+=== Create a Spring Boot Project
+
+Starting with Spring Initializr
+
+Navigate to https://start.spring.io. This service pulls in all the dependencies you need for an application and does most of the setup for you.
+
+Choose Maven and the Java language.
+
+Click Dependencies and select Spring Reactive Web.
+
+Click Generate.
+
+Download the resulting ZIP file, which is an archive of a web application that is configured with your choices.
+
+=== Create a Spring WebClient
+
+Creating a Spring REST WebClient would allow Spring application to communicate with the REST Endpoints exposed via Akka Serverless Service.
+
+cd into `akkaserverless-java-sdk/samples/java-valueentity-counter-spring-client`.
+
+Now you can create a `WebClient`. You need to configure the WebClient to use the Akka Serverless Service's host and port. The following listing (from `akkaserverless-java-sdk/samples/java-valueentity-counter-spring-client/src/main/java/com/example/client/spring/rest/config/WebClientConfig.java`) shows how to do so:
```

Comment:
We are already showing `src/main/java/com/example/client/spring/rest/config/WebClientConfig.java` below and we tell users to cd into `akkaserverless-java-sdk/samples/java-valueentity-counter-spring-client`, so maybe we can avoid mentioning it here as well. 

```suggestion
Now you can create a `WebClient`. You need to configure the WebClient to use the Akka Serverless Service's host and port. The following listing shows how to do so:
```

Reply:
Sure, that makes sense. I have removed these paths where they were used.